src/components/PokemonScreen/About/About.tsx

```diff
@@ -26,15 +29,15 @@ function About() {
                 style={{ transform: "rotate(90deg)" }}
               />
             </div>
-            <p>9,9 m</p>
+            <p>{pokemon.height} m</p>
           </div>
           <p className="attribute-name">Height</p>
         </div>
         <Divider orientation="vertical" />
         <div className="attribute">
           <div className="ability">
-            <p>Ability 1</p>
-            <p>Ability 2</p>
+            <p>{pokemon.abilities[0].ability.name}</p>
+            <p>{pokemon.abilities[1].ability.name}</p>
```

Comment:
```suggestion
           {pokemon.abilities.map((ability) => (
              <p key={ability.ability.name}>{ability.ability.name}</p>
            ))}
```
aqui tu poderia ter feito um map, alguns pokemons possuem mais de dois moves e outros apenas um, o que ocasionou em alguns cards nao abrirem